package.json

```diff
@@ -53,8 +53,8 @@
     "hoek": "^6.1.3",
     "joi": "^14.3.1",
     "serve": "^11.0.0",
-    "vuepress": "1.2.0",
-    "wait-for-localhost-cli": "^1.1.0"
+    "wait-for-localhost-cli": "^1.1.0",
+    "vuepress": "1.3.0"
```

Comment:
Did we need to update vuepress? If so, any concerns about updating to 1.3.0?

Reply:
yes in case we will keep the current shape of `on this page` component, most likely it's not required to update vuepress, so I will get rid of this part